### ISSUE TITLE
Missing pokrdb.dump

### ISSUE THREAD

**Issue by leegeng** [teampopong/pokr.kr#58]
pokrdb.dump 파일 다운로드 링크가 깨진 것 같습니다.

**Comment by jooddang** [comment]
네, 맞아요. 임시로 복구 시킨 파일을 새로운 테스트 서버에 올려놓고는 있는데 아직 data integrity 체크를 못했어요. 예전에 쓰던 서버를 받아서 데이터 cross check 를 해야 확실히 알 것 같은데, 이번달 말이나 다음달쯤 되어야 할 수 있을 것 같네요. (임대업체에게서 서버 넘겨받는 절차가 있어서..)
이슈 체크 고맙습니다!
